src/components/ProductDetails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+function Stars(props) {
+  let stars = props.stars;
+  let arr = [];
+  const totalStars = 5;
+  for (let i = 0; i < totalStars; i++) {
+    if (stars > 0) {
+      arr.push(<span className="glyphicon glyphicon-star"/>);
+      stars--;
+    } else {
+      arr.push(<span className="glyphicon glyphicon-star-empty"/>);
+    }
+  }
+  return (
+    <p>
+      {arr}
+    </p>
+  );
+}
+
+function ProductDetail(props) {
```

Comment:
Make sure to understand that the ProductDetail component is suppose to represent one single product.